.travis.yml

```diff
@@ -14,12 +14,12 @@ sudo: false
 
 cache:
   directories:
-    - vendor
```

Comment:
Why this change?

Reply:
Initially, it was done to get rid of broken vendor/installed.json which was the cause why every php 5.3.3 build was failing for no apparent reason, so it's not needed since the issue was resolved in composer and worked around here by updating composer to the latest snapshot.
I also thought it was redundant as long as composer cache was persisted across builds: cloning from cache should take nearly no time and it should be almost as effective as preserving vendor but saves some bytes for Travis :)
I totally can revert it if it feels wrong.